lib/drupalClient.ts

```diff
@@ -24,7 +29,27 @@ type PageData = {
     };
 }
 
-type MenuItem = {
+// Drupal menu item structure
+type DrupalMenuItem = {
```

Comment:
seems duplicated across files. is there no common types file or way of handling it? nuxt generated a shared types file